### ISSUE TITLE
Add example XML that can be directly round-tripped, with corresponding test

### ISSUE THREAD

**Issue by jmuhlich** [tlambert03/ome-types#47]
@tlambert03 I am considering reworking `testing/data/example.ome.xml` to use more (all?) of the schema elements and make sure all the float values etc. are round-trippable without tricks, then use that in a new test. Is that OK or do you think I should make a separate file? That file is ours, not from the ome-model test suite (it's example output from Ashlar).

**Comment by tlambert03** [comment]
Yep, sounds good. I was thinking of a similar test file that uses more of the schema.  Feel free to use that file

**Comment by jmuhlich** [comment]
Found some interesting issues around default values that we might actually want to emit to XML. The one I just hit was all the int attributes on `TiffData` that default to 0: `IFD`, `FirstC`, `FirstT`, and `FirstZ`. If you read this (imagine an OME-TIFF with two images, each with 3 channels):
```xml
<TiffData FirstC="0" FirstT="0" FirstZ="0" IFD="0">
<TiffData FirstC="1" FirstT="0" FirstZ="0" IFD="1">
<TiffData FirstC="2" FirstT="0" FirstZ="0" IFD="2">
...
<TiffData FirstC="0" FirstT="0" FirstZ="0" IFD="3">
<TiffData FirstC="1" FirstT="0" FirstZ="0" IFD="4">
<TiffData FirstC="2" FirstT="0" FirstZ="0" IFD="5">
```
Then our default-suppression would produce this output which looks a little lumpy:
```xml
<TiffData>
<TiffData FirstC="1" IFD="1">
<TiffData FirstC="2" IFD="2">
...
<TiffData IFD="3">
<TiffData FirstC="1" IFD="4">
<TiffData FirstC="2" IFD="5">
```
Obviously it will parse OK but it 's kind of unsatisfying. I propose we always emit these attributes even if they're set to the default of 0. (I actually think it's a mistake for the schema to have defined a default for these...) I checked and I didn't find anything else I would apply this treatment to. `Channel` and `Well` each have a `Color` int that defaults to -1 but that doesn't have the same issue as the TiffData stuff. I'll special-case these in `element_encode` for now, unless you have a better idea. If we ever end up with more fields to add to the list I'd refactor it at that point.
